lib/source-map-consumer.js

```diff
@@ -1139,20 +1038,21 @@ IndexedSourceMapConsumer.prototype._parseMappings =
     const originalMappings = this.__originalMappingsUnsorted = [];
     for (var i = 0; i < this._sections.length; i++) {
       var section = this._sections[i];
-      var sectionMappings = section.consumer._generatedMappings;
+
+      var sectionMappings = [];
+      section.consumer.eachMapping(m => sectionMappings.push(m));
+
       for (var j = 0; j < sectionMappings.length; j++) {
         var mapping = sectionMappings[j];
 
-        var source = section.consumer._sources.at(mapping.source);
-        source = util.computeSourceURL(section.consumer.sourceRoot, source, this._sourceMapURL);
+        var source = util.computeSourceURL(section.consumer.sourceRoot, source, this._sourceMapURL);
         this._sources.add(source);
         source = this._sources.indexOf(source);
 
         var name = null;
```

Comment:
I didn't examine more context but I suspect this line should be removed.

Reply:
The `var name = null;` line? The `name` variable is assigned to and used a little bit down this method.

Reply:
Thanks.